.devcontainer/node/devcontainer.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "Node",
+    "image": "mcr.microsoft.com/devcontainers/javascript-node:20",
+    "workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}/server/node",
+    "forwardPorts": [
+        3000
+    ],
+    "portsAttributes": {
+        "3000": {
+            "label": "Node",
+            "onAutoForward": "openBrowserOnce"
+        }
+    },
+    "secrets": {
+        "BRAINTREE_MERCHANT_ID": {
+            "description": "Sandbox merchant ID of the application.",
+            "documentationUrl": "https://www.braintreepayments.com/sandbox"
+        },
+        "BRAINTREE_PUBLIC_KEY": {
+            "description": "Sandbox public key of the application.",
+            "documentationUrl": "https://www.braintreepayments.com/sandbox"
+        },
+        "BRAINTREE_PRIVATE_KEY": {
+            "description": "Sandbox private key of the application.",
+            "documentationUrl": "https://www.braintreepayments.com/sandbox"
+        }
+    },
```

Comment:
I believe for SDK's, don't need to setup the BASE_URL. If environment is configured as sandbox, it will have all these configurations.